astro-docs/src/content/docs/reference/Powerpack/owners/Generators.mdoc

```diff
@@ -2,6 +2,8 @@
 title: '@nx/owners - Generators'
 sidebar:
   label: Generators
+weight: .5
+filter: 'type:References'
```

Comment:
short docs with exact matching titles soar to the top of the search results. so going for `init` results in this page being at the top. since the `init` generators are called for you automatically with `nx add` not super helpful doc page anyways. so toned it down.